changelogs/fragments/mm-feature__add-vcenter-host-connection.yml

```diff
@@ -0,0 +1,6 @@
+---
+minor_changes:
```

Comment:
The file name should start with the PR number 
the connects here can be combined into 1 and should start with the role name.